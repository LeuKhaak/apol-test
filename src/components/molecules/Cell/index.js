import React from "react";
import CellContent from "src/components/atoms/CellContent";
import * as styles from "./styles.module.scss";

function Cell({ content, cellStyle, mode, id }) {
  return (
    <div className={styles.cell}>
      <CellContent
        content={content}
        cellStyle={cellStyle}
        mode={mode}
        id={id}
      />
    </div>
  );
}

export default Cell;
