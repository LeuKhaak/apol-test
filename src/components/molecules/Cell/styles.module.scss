@import "../../../assets/styles/styles.scss";
.cell {
  @include cell;
}
