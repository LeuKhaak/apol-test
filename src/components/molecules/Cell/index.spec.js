import React from "react";
import { shallow } from "enzyme";
import Cell from ".";

let component = null;
const setComponent = (props) => shallow(<Cell {...props} />);
beforeEach(() => {
  component = setComponent();
});

describe("Cell testing:", () => {
  test("is Cell has style height", () => {
    component = setComponent({
      cellStyle: "cellTitleActions",
    });
    expect(component).toMatchSnapshot();
  });
});
