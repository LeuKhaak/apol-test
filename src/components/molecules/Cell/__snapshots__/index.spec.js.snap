// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell testing: is Cell has style height 1`] = `
<div
  style={
    Object {
      "height": "calc(100% - 1.2rem)",
    }
  }
>
  <CellContent
    cellStyle="cellTitleActions"
  />
</div>
`;
